system/modprobe.py

```diff
@@ -104,6 +107,13 @@ def main():
             if rc != 0:
                 module.fail_json(msg=err, **args)
             args['changed'] = True
+    elif args['state'] == 'probe':
+        if not present:
+            rc, _, err = module.run_command([module.get_bin_path('modprobe', True), args['name'], args['params']])
+            if rc == 0:
```

Comment:
Instead of a new probe state, this seems like it would be better as a probe option no/yes no by default that would work the same as the current present, which actually runs the same command.